jscomp/test/tagged_template_test.js

```diff
@@ -0,0 +1,43 @@
+// Generated by ReScript, PLEASE EDIT WITH CARE
+'use strict';
+
+var Mt = require("./mt.js");
+var Caml_splice_call = require("../../lib/js/caml_splice_call.js");
+var Tagged_template_libJs = require("./tagged_template_lib.js");
+
+function sql(prim0, prim1) {
+  return Caml_splice_call.spliceApply(Tagged_template_libJs.sql, [
+              prim0,
+              prim1
+            ]);
+}
```

Comment:
This shouldn't appear in the output since it isn't used.

Reply:
There should be a test file that calls the tagged template function as a function call to make sure that this helper is still included in that situation.

Reply:
I thought about this a bit more and it is actually being used because `sql` is exported.